docs/development/mono-repository.md

```diff
@@ -0,0 +1,95 @@
+---
+sidebar_position: 7
+---
+
+# Mono-Repository (monorepo) Tooling and Guidance
+
+Working across multiple repositories for the same project and/or team is cumbersome and slows down developer velocity. A way to improve developer velocity for teams and projects is to leverage a mono-repository (monorepo). With a monorepo, teams will be able to maintain multiple packages in a singular repository without having to switch to different repositories to maintain other packages.
+
+## Tooling
```

Comment:
Replace with `Recommended Packages`

Reply:
Explain monorepo may not be just javascript

Reply:
Monorepo could be scoped per team/project meaning any project could belong in a monorepo, or monorepo could be exclusively for a particular project.